moveit_ros/moveit_servo/include/moveit_servo/low_pass_filter.h

```diff
@@ -34,7 +34,7 @@
  * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
  * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-*******************************************************************************/
+ *******************************************************************************/
 
```

Comment:
Possible out of scope for this PR, but algorithm-heavy files like this would be better to live in moveit_core and only the ROS bindings are in moveit_ros. Would make these components much more reusable